hw/scripts/openocd.sh

```diff
@@ -101,7 +101,9 @@ openocd_debug () {
         if [ ! -z "$RESET" ]; then
             echo "mon reset halt" >> $GDB_CMD_FILE
         fi
-	echo "FILENAME" $FILE_NAME >>out
+  echo "$EXTRA_GDB_CMDS" >> $GDB_CMD_FILE
+  echo "FILENAME" $FILE_NAME >>out
```

Comment:
I don't see a reason for this line. Why is it here? Could it be removed as part of this commit?